# Patch to fix location bar focus KDE bugzilla 
# http://bugs.kde.org/show_bug.cgi?id=117031, Gentoo bug 114329.
--- konqueror/konq_mainwindow.cc.orig	2005-12-03 12:34:06.000000000 +0000
+++ konqueror/konq_mainwindow.cc	2005-12-03 12:34:44.000000000 +0000
@@ -3648,7 +3648,7 @@
 
 void KonqMainWindow::focusLocationBar()
 {
-  if ( m_combo->isVisible() )
+  if ( m_combo->isVisible() || !isVisible() )
     m_combo->setFocus();
 }
 
